Improves documentation about certificates auto-renewal enabling period (#734)
dnsimple/dnsimple-support: content/articles/letsencrypt.markdown

Let's Encrypt certificates are automatically renewed **30 days before the expiration date**, as suggested by Let's Encrypt, with automatic failover attempts every day in case of temporary failures.
</info>

> This feature is available for free to all accounts. You can enable/disable auto-renewal for a certificate from the SSL certificate page only if the certificate has not expired yet.

Reviewer: ```suggestion
This feature is available for free to all accounts. You can enable/disable auto-renewal for a certificate from the SSL certificate page. In order to use the feature the certificate must not be expired.
```
